faqs/galaxy/datasets_finding_datasets.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Finding Datasets
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+- To review all active Datasets in your account, go to **User > Datasets**.
+- Logging out of Galaxy while the Upload tool is still loading data can cause uploads to abort. This is most likely to occur when a dataset is loaded by browsing local files.
+- If you have more than one browser window open, each with a different Galaxy History loaded, the Upload tool will load data into the most recently used history.
+- Click on refresh icon {% icon galaxy-refresh %} at the top of the History panel to display the current active History with the datasets.
+
+*These are the possible ways to find your data if your dataset is missing, or you lost your datasets or your data dissapeared.*
```

Comment:
I think that we can remove this final sentence because it's already part of the title